<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CRecipeManagerClass</class>
 <widget class="QDialog" name="CRecipeManagerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>配方管理</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/CRecipeManager/Resources/recipe.png</normaloff>:/CRecipeManager/Resources/recipe.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>378</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>保存配方</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>配方切换</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBox">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QTableWidget" name="tableWidget_Camear">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>200</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="pushButton_remove">
     <property name="text">
      <string>移除配方</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CRecipeManager.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>comboBox</sender>
   <signal>activated(QString)</signal>
   <receiver>CRecipeManagerClass</receiver>
   <slot>SwitchRecipe(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>CRecipeManagerClass</receiver>
   <slot>SaveRecipe()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>832</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>854</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_remove</sender>
   <signal>clicked()</signal>
   <receiver>CRecipeManagerClass</receiver>
   <slot>RemoveRecipe()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>712</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>718</x>
     <y>-7</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SwitchRecipe(QString)</slot>
  <slot>SaveRecipe()</slot>
  <slot>RemoveRecipe()</slot>
 </slots>
</ui>
